tests/functional/graphql/test_related.py

```diff
@@ -0,0 +1,203 @@
+                                                             "b4cbf203-50cd-45fc-97c8-e1fddac2f270"}
+
+    def test_related_end_of_article_article_doesnt_exist(self):
+        with TestClient(app) as client:
+            response = client.post("/", json={
+                    'query': '''
+                        query ($representations: [_Any!]!) {
+                          _entities(representations: $representations) {
+                            ... on SyndicatedArticle {
+                                itemId
+                                publisherUrl
+                                relatedEndOfArticle(count: 3) {
+                                    id
+                                    corpusItem {
+                                      id
+                                    }
+                                }
+                            }
+                          }
+                        }
+                    ''',
+                    'variables': {
+                        'representations': [
+                            {
+                                '__typename': 'SyndicatedArticle',
+                                'itemId': "11111",
+                                'publisherUrl': "xxxxx",
+                            },
+                        ],
+                    },
+                }).json()
+
+            assert not response.get('errors')
+            recs = response['data']['_entities'][0]['relatedEndOfArticle']
+            assert len(recs) == 0
+
+    def test_related_right_rail_article_doesnt_exist(self):
+        with TestClient(app) as client:
+            response = client.post("/", json={
+                'query': '''
+                        query ($representations: [_Any!]!) {
+                          _entities(representations: $representations) {
+                            ... on SyndicatedArticle {
+                                itemId
+                                publisherUrl
+                                relatedRightRail(count: 3) {
+                                    id
+                                    corpusItem {
+                                      id
+                                    }
+                                }
+                            }
+                          }
+                        }
+                    ''',
+                    'variables': {
+                        'representations': [
+                            {
+                                '__typename': 'SyndicatedArticle',
+                                'itemId': "111111",
+                                'publisherUrl': "xxxx",
+                            },
+                        ],
+                    },
+                }).json()
+
+            assert not response.get('errors')
+            recs = response['data']['_entities'][0]['relatedRightRail']
+            assert len(recs) == 0
```

Comment:
Nit: Configure PyCharm to automatically add a newline to the end of files in Editor > General > 'Ensure every saved file ends with a line break'.